### PR TITLE
feat: update resource pool cards [DET-6417]

### PR DESCRIPTION
**Pull request by gt2345**
## Description

Update resource pool cards.
New field added:
- Accelerator
- queued

## Test Plan

Navigate to `/clusters` page with different type of resource pools, with different states of running/queued experiments

Design file:
https://www.figma.com/proto/MdZBYvR0KJ1uBDjx203IAv/Cluster-Consolidation?page-id=2%3A139&node-id=412%3A6909&viewport=268%2C48%2C1&scaling=scale-down-width

Before:
Screen Shot 2022-03-06 at 8 00 12 PM

After:
Screen Shot 2022-03-06 at 8 00 26 PM

### CODE REVIEW

File: webui/react/src/components/SlotAllocationBar.tsx [hunk 1]
```diff
@@ -148,6 +160,29 @@ const SlotAllocationBar: React.FC<Props> = ({
           <Bar {...barProps} parts={barParts} />
         </div>
       </ConditionalWrapper>
+      {footer && (
+        <div className={css.footer}>
+          {poolType === V1ResourcePoolType.K8S ? (
+            <header>{`${footer.isAux ?
+              `${footer.auxRunning} Aux Container Running` :
```
Comment (mapmeld) [inline webui/react/src/components/SlotAllocationBar.tsx:167]: In this and line 173 using `footer.auxRunning`, should be <number> Aux Container**s** Running

Comment (mapmeld) [inline webui/react/src/components/SlotAllocationBar.tsx:167]: In this and line 173 using `footer.auxRunning`, should be <number> Aux Container**s** Running

**Review by mapmeld — APPROVED** [review]
I can only test on my local CPU, but everything lines up with what we have on the old Cluster view.

File: webui/react/src/components/SlotAllocationBar.tsx (hunk 1 above)
Comment (mapmeld) [reply webui/react/src/components/SlotAllocationBar.tsx]: ^^ can you make these two edits to the text?

Comment (gt2345) [reply webui/react/src/components/SlotAllocationBar.tsx]: I see! Sorry I misunderstood the first time!

File: webui/react/src/components/Bar.tsx [hunk 2]
```diff
@@ -7,6 +7,7 @@ import { floatToPercent } from 'utils/string';
 import css from './Bar.module.scss';
 
 export interface BarPart {
+  borded?: string;
```
Comment (ioga) [inline webui/react/src/components/Bar.tsx:10]: should this be `bordered`?

File: webui/react/src/components/Bar.tsx [hunk 3]
```diff
@@ -19,10 +20,19 @@ export interface Props {
...
-  return {
+  let style = {
     backgroundColor: part.color,
+    borderColor: 'var(--theme-colors-monochrome-11)',
+    borderStyle: 'none',
+    borderWidth: 1,
     width: floatToPercent(part.percent, 0),
   };
+
+  if(part.borded){
```
Comment (ioga) [inline webui/react/src/components/Bar.tsx:31]: hmm, why isn't lint erroring out on this line?
overwhelming majority of our code has a space in `if (` - I believe we should be enforcing this

File: webui/react/src/fixtures/responses/cluster/resource-pools.json [hunk 4]
```diff
@@ -3,6 +3,7 @@
         "name": "perception-cpu-aws",
         "description": "AWS CPU-only resource pool for tensorboard AWS CPU-only resource pool for tensorboard AWS CPU-only resource pool for tensorboards AWS CPU-only resource pool for tensorboards AWS CPU-only resource pool for tensorboards",
         "type": "RESOURCE_POOL_TYPE_AWS",
+        "accelerator": "",
```
Comment (ioga) [inline webui/react/src/fixtures/responses/cluster/resource-pools.json:6]: should these fixtures include QueueStats data?

**Review by ioga — APPROVED** [review]

**Pull request merged**